ci_tutorial/products_pipeline/distributed_build.rst

```diff
@@ -0,0 +1,113 @@
+                    "packages": [
+                        [
+                            {
+                                "package_id": "de738ff5d09f0359b81da17c58256c619814a765",
+                                "binary": "Build",
+                                "build_args": "--requires=engine/1.0 --build=engine/1.0",      
+                            }
+                        ]
+                    ]
+                }
+            ],
+            [
+                {
+                    "ref": "game/1.0#1715574045610faa2705017c71d0000e",
+                    "depends": [
+                        "engine/1.0#fba6659c9dd04a4bbdc7a375f22143cb"
+                    ],
+                    "packages": [
+                        [
+                            {
+                                "package_id": "bac7cd2fe1592075ddc715563984bbe000059d4c",
+                                "binary": "Build",
+                                "build_args": "--requires=game/1.0 --build=game/1.0",
+                            }
+                        ]
+                    ]
+                }
+            ]
+        ]
+    }
+
+
+For convenience, in the same way that ``conan graph info ... --format=html > graph.html`` can generate a file with an HTML interactive dependency graph, the ``conan graph build-order ... --format=html > build_order.html`` can generate an HTML visual representation of the above json file:
+
+
+.. image:: ./build_order_simple.png
+   :width: 500 px
+   :align: center
+
+
+The resulting json contains an ``order`` element which is a list of lists. This arrangement is important, every element in the top list is a set of packages that can be built in parallel because they do not have any relationship among them. You can view this list as a list of "levels", in level 0, there are packages that have no dependencies to any other package being built, in level 1 there are packages that contain dependencies only to elements in level 0 and so on.
+
+Then, the order of the elements in the top list is important and must be respected. Until the build of all the packages in one list item has finished, it is not possible to start the build of the next "level".
+
+Using the information in the ``graph_build_order.json`` file, it is possible to execute the build of the necessary packages, in the same way that the previous section ``--build=missing`` did, but not directly managed by us.
+
+Taking the arguments from the json, the commands to execute would be:
+
+.. code-block:: bash
+
+    $ conan install --requires=engine/1.0 --build=engine/1.0
+    $ conan install --requires=game/1.0 --build=game/1.0
+
+We are executing these commands manually, but in practice, it would be a ``for`` loop in CI executing over the json output. We will see some Python code later for this. At this point we wanted to focus on the ``conan graph build-order`` command, but we haven't really explained how the build is distributed.
```

Comment:
Maybe some re-ordering of these parts is in order, since we name it `distributed build` but do not in fact explain it

Reply:
I have renamed this section, now it is called "the build-order", which I think is better than "distributed build", because it is a bit too early to introduce that concept here, and it is delayed until last scenario.